linera-chain/src/manager.rs

```diff
@@ -108,6 +108,35 @@ pub enum Outcome {
 
 pub type ValidatedOrConfirmedVote<'a> = Either<&'a Vote<ValidatedBlock>, &'a Vote<ConfirmedBlock>>;
 
+/// The current locked block: Validators are only allowed to sign any proposal with a different
+/// block if they see a `ValidatedBlock` certificate with a higher round.
```

Comment:
Do you mean:
_Validators are allowed to sign a different block (from the locked block) iff they see a `ValidatedBlockCertificate` for it with a higher round_

if so, consider replacing the comment as my version reads better (IMHO ofc).